javascript-modal/main.js

```diff
@@ -0,0 +1,15 @@
+var $body = document.querySelector('body');
+var $modalBackground = document.querySelector('.modal-background');
+var $modalNo = document.querySelector('.no');
+$body.removeChild($modalBackground);
+var $openModal = document.querySelector('.open-modal');
+function modalOn(event) {
+  $body.appendChild($modalBackground);
```

Comment:
Instead of repeatedly appending and removing the child (which is a pretty inefficient action for the browser), you should make a class ruleset in CSS to hide an element, then add and remove that class as necessary to hide or show the modal.

Reply:
wow... that's definetly less work, xD